rs/bitcoin/ckbtc/minter/src/state/eventlog.rs

```diff
@@ -367,7 +367,7 @@ pub fn replay<I: CheckInvariants>(
                 account,
                 reason,
             } => {
-                state.suspended_utxos.insert(account, utxo, reason);
+                state.suspended_utxos.insert(account, utxo, reason, None);
```

Comment:
Does it mean after each upgrade all suspended_utxos can be re-evaluated immediately?

Reply:
That's correct (correspond to the current state in master without timestamps). I thought this is acceptable (please feel free to challenge):

1. There are currently only 68 suspended UTXOs, most of them (65) being ignored due to having a value inferior to the current KYT fee.
2. There shouldn't be additional ignored UTXOs once we migrate to the bitcoin checker, because the checker fee is less than the dust limit.
3. I would apriori expect less quarantined UTXOs than now (because it's based on some static list of like 500 addresses instead of some dynamic check)

WDYT?

Reply:
Thanks for the explanation! LGTM!